### ISSUE TITLE
Save screenshot values in ini file

### ISSUE THREAD

**Issue by myhobby2** [E2OpenPlugins/e2openplugin-OpenWebif#1447]
**Is your feature request related to a problem? Please describe.**

I use live screenshot to remotely check my boxes. But every time I need to tick:
- High Resolution 
- Refresh automatically every 
- xx seconds 

**Describe the solution you'd like**
It would be very nice to store your preferences in a config file a default values

**Describe alternatives you've considered**
As far as I investigated, there is none. Manual entry every time is my "workaround"

**Additional context**
Please ask if request/info is not clear .

**Comment by jbleyel** [comment]
The settings are stored in the local storage of the browser like a cookie.
This should be supported by all browsers.

**Comment by myhobby2** [comment]
The cookie solution is clear but not workable if you work on multiple different devices and when cookies are frequently deleted. Would be nice if this could be set on the box

**Comment by wedebe** [comment]
I agree with @myhobby2, it would make for a consistent cross-device experience. In addition I'd like to have image size as a dropdown.

**Comment by jbleyel** [comment]
HI @wedebe , please test -> https://github.com/E2OpenPlugins/e2openplugin-OpenWebif/tree/screenshot

**Comment by myhobby2** [comment]
Does this mean that my feature requests remains open ? Not clear to me at the moment

**Comment by jbleyel** [comment]
No .. the request is now implemented

**Comment by myhobby2** [comment]
Thanks for that. I would like to test this on my Enigma2 image (Openpli 7), but I'm not qualified enough to update my box based on the github adaptions. Can this be wrapped in a ipk or do you have an instruction how to update my OpenWebif version on these boxes

**Comment by jbleyel** [comment]
Please ask the PLi Team to make the ipk for you or install this.

https://github.com/E2OpenPlugins/e2openplugin-OpenWebif/raw/gh-pages/enigma2-plugin-extensions-openwebif_latest_all.ipk

**Comment by myhobby2** [comment]
Top! I can install this ipk. Fingers crossed and will provide feedback
